addons/binding/org.openhab.binding.max/src/main/java/org/openhab/binding/max/internal/message/H_Message.java

```diff
@@ -104,22 +106,26 @@ public String getNTPCounter() {
     }
 
     private final void setDateTime(String hexDate, String hexTime) {
-
-        int year = Utils.fromHex(hexDate.substring(0, 2));
+        // we have to add 2000, otherwise we get a wrong timestamp
+        int year = 2000 + Utils.fromHex(hexDate.substring(0, 2));
```

Comment:
I do not like this. Ideas are appreciated.

Reply:
I do not really have any other idea